TheBTeam.Web/Views/CategoryBudget/EmptyList.cshtml

```diff
@@ -8,15 +8,14 @@
 <h2>There is no budget for @CultureInfo.CurrentCulture.DateTimeFormat.GetMonthName(ViewBag.Date.Month) of @ViewBag.Date.Year</h2>
 
 <h3><a asp-action="Create" asp-route-id="@ViewBag.Id" asp-route-date="@ViewBag.Date">Create</a> one or select another date</h3>
-<div class="form-group">
-    <input asp-controller="CategoryBudget" asp-action="Create" type="submit" value="Create" class="btn btn-primary"  asp-route-id="@ViewBag.Id" asp-route-date="@ViewBag.Date" >
-</div>
+
```

Comment:
I might be doing something horrobly wrong,

Reply:
okay, quick update, it doesnt work for santiago which's details ive previously edited, its ok for sofia